.drone-v1.yml

```diff
@@ -0,0 +1,248 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    environment:
+      SPRING_PROFILES_ACTIVE: 'development, local'
+      SERVER_PORT: '8080'
+      SEARCH_QUEUE_NAME: 'search-queue'
+      SEARCH_QUEUE_DLQ_NAME: 'search-queue-dlq'
+      DB_HOST: 'postgres'
+      AWS_LOCAL_HOST: 'localstack'
+    commands:
+      - ./gradlew build
```

Comment:
Split this into two steps: `assemble` and `check`.

Reply:
This has been completed